proposals/wg-reference-spec.md

```diff
@@ -0,0 +1,59 @@
+# OCI Working Group: Reference Spec
+
+# Reference Spec OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the Reference Spec Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”). The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter. Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+References are a string that is used by runtimes and other OCI registry clients to retrieve a container image.
+They are currently a convention, used by many clients, adopted from Docker and implemented in distribution/distribution.
+This WG seeks to define the syntax and parsing of a reference as an OCI spec.
+
+## Scope
```

Comment:
Does this cover the reference string for local OCI archives?

Reply:
Is that [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md)? I've updated this with a link to make it clear that's what I meant by OCI Layout.